style.css

```diff
@@ -414,352 +398,332 @@ form button:active {
   box-shadow: none;
 }
 
-/* Mobile */
-@media only screen and (max-width: 767px) {
-  .nav-links {
-    display: none;
-  }
+#cards-desktop {
+  display: none;
+}
 
-  #portfolio-container {
-    display: none;
-  }
+.lang-lang-desktop {
+  display: none;
 }
 
-/* Desktop */
-@media only screen and (min-width: 768px) {
-  main {
-    padding-left: 250px;
-    background-image: url(./images/contact\ header-shapes\ mobile.svg);
-    background-size: cover;
-    background-repeat: no-repeat;
-  }
+#close-menu-btn {
+  display: none;
+}
 
-  .logo {
-    padding-left: 30px;
-  }
+#mobile-nav {
+  margin-bottom: -32px;
+  color: #6070ff;
+}
 
-  .nav-bar .logonav {
-    margin-left: 300px;
-  }
+.nav__container {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+  background-color: white;
+  position: relative;
+}
 
-  .menu img {
-    display: none;
-  }
+.nav__menu {
+  margin-top: -1rem;
+  right: 3%;
+  flex-direction: column;
+  gap: 1rem;
+  position: fixed;
+  left: 0;
+  width: 100%;
+  height: 80%;
+  background: rgba(26, 84, 231, 0.8);
+  display: none;
+  overflow: auto;
+}
```

Comment:
- [ ] Please kindly ensure that the mobile nav menu covers the entire length of the screen as it is in the design.
- [ ] Please also ensure that the spacing between the nav list items(Portfolio, About, Contact) is implemented according to the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/175240899-49e899b2-4ef7-4966-9a67-2ae9bf05bedc.png)


### Original Design
![image](https://user-images.githubusercontent.com/43995830/175240769-620d9b89-8b02-4f39-bb53-d549b8f2dda3.png)